governance/votes/Executive vote - September 27, 2023.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Template - [Executive Vote] DAO Resolution for Monetalis Clydesdale, DAO Resolution for HV Bank, Fortunafi vault changes, Spark Protocol Proxy Spell - September 27, 2023
+summary: Doc variable update including DAO Resolution for Monetalis Clydesdale (RWA007-A); doc variable update including DAO Resolution for HV Bank (RWA009-A); place Fortunafi (RWA005-A) vault into soft liquidation state; trigger Spark Protocol Proxy Spell.
+date: 2023-09-27T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] DAO Resolution for Monetalis Clydesdale, DAO Resolution for HV Bank, Fortunafi vault changes, Spark Protocol Proxy Spell - September 27, 2023
+
+The Governance Facilitators, Dewiz, Sidestream and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+- The Monetalis Clydesdale (RWA007-A) vault [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated, to include the latest DAO Resolution as detailed below.
+- The HV Bank (RWA009-A) vault [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated, to inlcude the latest DAO Resolution as detailed below.
+- Fortunafi (RWA005-A) vault will be placed into soft liquidation state, as detailed below.
+- A Spark Protocol Proxy Spell will be triggered that activates the Gnosis Chain instance of Spark Lend, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Monetalis Clydesdale (RWA007-A) Vault Doc Update
```

Comment:
Suggest "RWA Agreement Parameter" Update - per LFW's advice to use human friendly language :)

If you agree, change for HV Bank header as well.